README.md

```diff
@@ -2,140 +2,167 @@
+                reqs=["torch", "diffusers", "~/code/my_other_repo"],
+                setup_cmds=["source activate ~/.bash_rc"],
+                env_vars={"TOKEN": "1234"},
+                workdir="./")  # Current git root
+my_env = my_env.to(gpu)  # Called implicitly if passed to Function.to, like above
 ```
 
-Reuse your saved compute and data resources from anywhere, with a single line of Python.
+rh.Function is actually a special case of rh.Module, which is how we send classes to clusters. There are a number of
+other built-in Modules in Runhouse, such as Blob, Queue, KVstore, and more. Modules are very flexible. You can
+leave them as classes or create instances, send the class or the instances to clusters, and even create instances
+remotely. You can call class methods, properties, generators, and async methods remotely, and they'll behave 
+the same as if they were called locally (e.g. generators will stream, asyncs will return awaitables), including 
+streaming logs and stdout back to you.
 
 ```python
-sd_generate = rh.function(name="sd_generate")
-image = sd_generate("A hot dog made of matcha.")
-
-folder_on_s3 = rh.folder(name="dreambooth_outputs")
-folder_on_local = folder_on_s3.to("here")
+# You can create a module out of an existing class 
+MyRemoteClass = rh.module(MyClass).to(gpu)
+MyRemoteClass.my_class_method(1, 2, 3)
+# Notice how we sent the module to gpu above, so now this instance already lives on gpu
+my_remote_instance = MyRemoteClass(1, 2, 3)
+
+# You can define a new module as a subclass of rh.Module to have more control 
+# over how it's instantiated, or provide it within a library
+class MyCounter(rh.Module):
+    def __init__(self, count, **kwargs):
+        super().__init__(**kwargs)
+        self.count = count
+
+    def increment(self, y):
+        self.count += y
+        print(f"New count: {self.count}")
+        return self.count
+
+my_remote_counter = MyCounter(count=1).to(gpu, name="my_counter")
+my_remote_counter.increment(2)  # Prints "New count: 3" and returns 3
 ```
 
-These APIs work from anywhere with a Python interpreter and an internet connection.
-Notebooks, scripts, pipeline nodes, etc. are all fair game.
-
-### 🙅‍♀️ Runhouse is not
-
-* An orchestrator / scheduler (Airflow, Prefect, Metaflow)
-* A distributed compute DSL (Ray, Spark, Dask)
-* An ML Platform (Sagemaker, Vertex, Azure ML)
-* A model registry / experiment manager / MLOps framework (MLFlow, WNB, Kubeflow)
-* Hosted compute (Modal, Banana, Replicate)
-
-## 🐣 Getting Started
-
-Please see the [Getting Started guide](https://www.run.house/docs/tutorials/quick_start).
-
-tldr;
-```commandline
-pip install runhouse
-# Or "runhouse[aws]", "runhouse[gcp]", "runhouse[azure]", "runhouse[all]"
-
-# [optional] to set up cloud provider secrets:
-sky check
-
-# [optional] login for portability:
-runhouse login
+You can also call the Runhouse HTTP server directly (though you may need to open a port or tunnel to do so):
```

Comment:
let's just drop this for now?

Reply:
The code here actually does work, and the point is for people to understand that this is a real service and not exclusively available through a Python client. Does that make sense?

Reply:
Are you saying we should get rid of it in favor of when we can advertise opening ports with Den Auth enabled (or proxying) more confidently?